README.md

```diff
@@ -58,6 +58,24 @@ To do the initial setup and ensure that all tests pass locally:
 make
 ```
 
+#### Feature Tests
+
+Setup steps:
+
+1. Edit _.env_ file
+  * *API\_REDIRECT*=__http://web:8080__
+  * *API\_BASE\_URL*=__http://api:3000__
+1. Disable *SAML\_ENABLED* auth and enable *BASIC\_ENABLED*
+1. OPTIONAL Add _web_ and _api_ to the end of the localhost line in /etc/hosts so you don't have to keep changing the .env file
```

Comment:
I'm not using Docker on Mac, but by saying it is optional do you mean that even without adding `web` and `api` to `/etc/hosts`, the host OS will be able to route to `web:8000` and `api:3000`?